### ISSUE TITLE
Unquoted command substitution in _fasd_prompt_func results in "no match" errors in bash when the failglob option is set in bash

### ISSUE THREAD

**Issue by zmwangx** [clvv/fasd#73]
This issue should be easy to reproduce:

```bash
> eval "$(fasd --init auto)"
> shopt -s failglob
> echo crazysh*t
bash: no match: crazysh*t
bash: no match: crazysh*t
> echo 'crazysh*t'
crazysh*t
bash: no match: 'crazysh*t'
```

Note how the `no match` error message is duped after `echo crazysh*t`, and how we get a `no match` error even after a perfectly legit `echo 'crazysh*t''.

The problem can be traced to `_fasd_prompt_func`:

```bash
> type _fasd_prompt_func
_fasd_prompt_func () {
    eval "fasd --proc $(fasd --sanitize $(history 1 |     sed "s/^[ ]*[0-9]*[ ]*//"))" >> "/dev/null" 2>&1
}
```

Note how `$(history 1 |     sed "s/^[ ]*[0-9]*[ ]*//")` could evaluate to practically anything, yet left unquoted, which is the source of the `no match` errors.

**I suggest we simply wrap this command substitution in quotes: `"$(history 1 |     sed "s/^[ ]*[0-9]*[ ]*//")"`**. This shouldn't have any side effect. After all, it doesn't matter whether `$(history 1 |     sed "s/^[ ]*[0-9]*[ ]*//")` is a single word or multiple words, because `fasd --sanitize` will join them together with `"$*"` anyway.

**Comment by clvv** [comment]
Yes, I think you have the right analysis and fix here. Thanks for submitting this! If you create a pull request, I will just merge this in.
